patches/0009-LPS-127732-remove-css-classes-as-well-when-changing-.patch

```diff
@@ -0,0 +1,22 @@
+From 213029994cdb8f07eb8299d94a94aeab314c958b Mon Sep 17 00:00:00 2001
+From: Norbert Nemeth <norbert.nemeth@liferay.com>
+Date: Thu, 11 Mar 2021 16:09:52 +0100
+Subject: [PATCH] LPS-127732 remove css classes as well when changing style
+
+---
+ plugins/removeformat/plugin.js | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/plugins/removeformat/plugin.js b/plugins/removeformat/plugin.js
+index e1e41bb3e..8aa04768a 100644
+--- a/plugins/removeformat/plugin.js
++++ b/plugins/removeformat/plugin.js
+@@ -35,7 +35,7 @@ CKEDITOR.plugins.removeformat = {
+ 					range;
+ 
+ 				while ( ( range = iterator.getNextRange() ) ) {
+-					range.enlarge( CKEDITOR.ENLARGE_INLINE );
++					range.enlarge( CKEDITOR.ENLARGE_ELEMENT );
```

Comment:
I don't really understand _how_ this fixes the bug, but perhaps @julien does. Are you able to break it down for us @NemethNorbert?

Reply:
Hey @wincent,

**range** represents a delimited piece of content in a DOM Document. It is contiguous in the sense that it can be characterized as selecting all of the content between a pair of boundary-points.
**range.enlarge()** Expands the range so that partial units are completely contained. It relieves two parameters:
The unit type to expand with. Use one of following values: CKEDITOR.ENLARGE_BLOCK_CONTENTS, CKEDITOR.ENLARGE_ELEMENT, CKEDITOR.ENLARGE_INLINE, CKEDITOR.ENLARGE_LIST_ITEM_CONTENTS.
and
[ excludeBrs ] : Boolean
Whether include line-breaks when expanding.

As far as I understood the documentation, by using **CKEDITOR.ENLARGE_ELEMENT**, I basically expand the selection to the whole element, that will also include the **class** attribute of the html element.

resources:
https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dom_range.html#method-enlarge
https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR.html#property-ENLARGE_ELEMENT
https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR.html#property-ENLARGE_INLINE

Thanks

Reply:
Wonderful explanation. Thanks @NemethNorbert! Next time consider including it in your commit message (which is used to seed the PR description, so we'd see it there too).

I think we can merge this.